src/actions/curationActions.js

```diff
@@ -2,7 +2,8 @@
 // SPDX-License-Identifier: MIT
 
 import React from 'react'
-import { flatten } from 'lodash'
+import flatten from 'lodash/flatten'
```

Comment:
The main advantage is on the final bundle size, using this method only the required module will be really imported by webpack.
You can see here a test of performances on this specific behaviour: https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark

Reply:
fair enough. For this style to be effective, do we need to do this everywhere? If so, let's be consistent until we decide to do that optimization across the entire codebase. 

Consistency wins! Otherwise the code gets really messy.

Reply:
@jeffmcaffer
Sorry to jump in in this but I want to make sure we are all on the same page.

While I do agree that Consistency wins, I do also think that if we want to introduce improvements, best practices, etc. it could be beneficial to "lead by example" and optimize new code, while being mindful of avoiding "messy code"

We could then provide evidence of th3 benefit and address refactoring for existing code.

Given the nature of the lodash change, may i suggest to bring it in?

Otherwise, let us know your thoughts and how do you think we should approach the introduction of new best practices, refactoring etc from a process perspective. 

Thanks!

Reply:
What I want to avoid is that good ideas for programming style come up, get used in a bit of the code that is touched and then never ends up spreading. That leaves us with an inconsistent codebase and no tangible improvements. 

To execute on this I see two approaches. 

1. Augment the coding practices doc to detail the choices and explicitly call out that they will be incrementally introduced. Ideally with some sort of target time and commitment to get it done.
2. Do a full on sweep of the code and just change it all at once. Would be good to time those things for a time when it won't generate a lot of merge conflicts but sure.

This is especially true of changes that need to be pervasive to be effective. 

Either is fine with me but honestly at this point optimizing out a few K of lodash is way low on the priority list. Even as a general practice, until we see that load time is an issue (eg., when we go to support mobile) to me consistency still wins.

Reply:
@jeffmcaffer Thanks for your repy!

I couldn't agree more with all of the above and agree on the fact that the lodash change is very low priority. 


As for the process, I think we all agree. 

Extending the documentatiom definitely make sense when introducing coding pratices and styleguides, probably less when tuning a very specific bit of code.

And yes, we will make sure to be consistent, and make sure to apply improvents anywhere needed at once, as long as that doesn't cause huge merge conflicts or has a big impact on timelines etc.